### PR TITLE
Adding columns for: units, dividends, PnL

### PR DESCRIPTION
**Pull request by p-oz**
Added some code to include columns for UNITs, Dividends and PnL and also to handle multiple cash accounts with different currencies

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -157,18 +175,65 @@ def _account_metadata_pattern(self, tree, metadata_key, pattern, internal, mwr,
...
+            row['Dividends'] = self._process_dividends(node,row_currency)
+
         if self.operating_currency in balance and self.operating_currency in cost:
             balance_dec = round(balance[self.operating_currency], 2)
             cost_dec = round(cost[self.operating_currency], 2)
             row["balance"] = balance_dec
             row["cost"] = cost_dec
+        
+        #### ADD other Currencies
+        elif self.other_currencies[0] in balance and self.other_currencies[0] in cost:
```
Comment (PhracturedBlue) [inline __init__.py]: self.other_currencies could be an empty set, you can't always access element 0 from it

File: templates/PortfolioSummary.html [hunk 2]
```diff
@@ -1,41 +1,66 @@
...
+  .tree-table .num .date {
+    font-style: italic;
+    font-size: x-small;
+  }
+</style>
+
 <h2>Portfolio Summary Report</h2>
 <br />
 {% for title, data in extension.portfolio_accounts() %}
-  {% set types = data[0] %}
```
Comment (PhracturedBlue) [inline templates/PortfolioSummary.html:7]: please don't change the indentation.  It makes it harder to review and run 'git annotate' on.  Also, I liked it the way it was

File: __init__.py [hunk 3]
```diff
@@ -157,18 +175,65 @@ def _account_metadata_pattern(self, tree, metadata_key, pattern, internal, mwr,
...
+
         row["account"] = node.name
         row['children'] = []
+        row["last-date"] = None
+        row['PnL'] = ZERO
+        row['Dividends'] = ZERO
         date=self.ledger.end_date
         balance = cost_or_value(node.balance, "at_value", g.ledger.price_map, date=date)
         cost = cost_or_value(node.balance, "at_cost", g.ledger.price_map, date=date)
+
```
Comment (PhracturedBlue) [inline __init__.py]: Please remove all trailing whitespace.

**Review by PhracturedBlue — CHANGES_REQUESTED** [review]
Please run pylint on the code and fix all issues.

**Comment by p-oz** [comment]
Done and fixed

**Comment by PhracturedBlue** [comment]
Thanks, Merged

**Pull request closed without merging**